Submitted this upstream and should be in the next release.

commit a89016726656776c42643a021c09ce139b0c196e
Author: Gerald Pfeifer <gerald@pfeifer.com>
Date:   Thu Jul 22 09:51:02 2021 +0000

    nsiproxy: Include sys/socket.h

index ab1b6294bea..93fdefa24cc 100644
--- dlls/nsiproxy.sys/ip.c
+++ dlls/nsiproxy.sys/ip.c
@@ -21,6 +21,10 @@
 #include "config.h"
 #include <stdarg.h>
 
+#ifdef HAVE_SYS_SOCKET_H
+#include <sys/socket.h>
+#endif
+
 #ifdef HAVE_NETINET_IN_H
 #include <netinet/in.h>
 #endif
